### ISSUE TITLE
Icon tool tip

### ISSUE THREAD

**Issue by ShlomiRosolio** [AlaskaAirlines/auro-icon#50]
## Describe the bug
There is an unwanted tooltip on hover. The tooltip text cannot be changed or removed.

## To Reproduce
1. Go to https://auro.alaskaair.com/components/auro/icon
2. Hover on the first icon

## Expected behavior
No tooltip should popup unless specified.

## Screenshots
![Screen Shot 2022-09-15 at 12 42 08 PM](https://user-images.githubusercontent.com/89954406/190461029-9d22fbe7-4ff1-47b6-a616-d594fef5d287.png)

**Comment by blackfalcon** [comment]
No further action was taken.
